curriculum/challenges/english/10-coding-interview-prep/rosetta-code/comma-quibbling.md

```diff
@@ -8,29 +8,18 @@ dashedName: comma-quibbling
 
 # --description--
 
-Comma quibbling is a task originally set by Eric Lippert in his [blog](https://blogs.msdn.com/b/ericlippert/archive/2009/04/15/comma-quibbling.aspx).
+Comma quibbling is a task originally set by Eric Lippert in his 2009 blog. In this challenge, you will create a `string` from an `array`. You have to account for the `array` having none items, single item, or multiple items in it. 
```

Comment:
```suggestion
Comma quibbling is a task originally set by Eric Lippert in his 2009 blog. In this challenge, you will create a `string` from an `array`. You have to account for the `array` having no items, a single item, or multiple items in it. 
```

Reply:
I will look back to the old one and try to implement it back